src/test/groovy/io/openliberty/tools/gradle/TestLooseApplication.groovy

```diff
@@ -87,16 +89,24 @@ public class TestLooseApplication extends AbstractIntegrationTest{
 
       // Check that dependencies are located in the test build dir specified by copyLibsDirectory. Otherwise they would be located in the gradle cache somewhere.
       String nodeValue = nodes.item(0).getAttributes().getNamedItem("sourceOnDisk").getNodeValue();
-      assert nodeValue.endsWith("/commons-text-1.1.jar") && nodeValue.contains("/test-loose-application/build/libs/") : 'archive sourceOnDisk attribute value not correct'
 
+      if (OSUtil.isWindows()) {
+          Assert.assertTrue('archive sourceOnDisk attribute value not correct', nodeValue.endsWith("\\commons-text-1.1.jar") && nodeValue.contains("\\test-loose-application\\build\\libs\\"))
+      } else {
+          Assert.assertTrue('archive sourceOnDisk attribute value not correct', nodeValue.endsWith("/commons-text-1.1.jar") && nodeValue.contains("/test-loose-application/build/libs/")) 
+      }
 
       Assert.assertEquals("archive targetInArchive attribute value", "/WEB-INF/lib/commons-lang3-3.5.jar",
```

Comment:
Why doesn't this comparison need to be OS specific?

Reply:
Seems that the targetInArchive values use the same formatting regardless of the OS. I'd guess that's just due to how the plugins build those paths rather than being something we get back from the OS/Gradle/Java like the other values we're checking that are OS dependent.

Reply:
Answering my own question....`targetInArchive` is relative to the contents of the archive...those always have forward slashes...it is expected.